conf/application.conf

```diff
@@ -24,9 +24,10 @@ play {
   }
   filters.headers {
     # Unset some of the security filters enabled in datastore.Filters
-    contentSecurityPolicy = null
+    contentSecurityPolicy = "script-src 'self'"
```

Comment:
the same should probably be added to standalone-datastore.conf and standalone-tracingstore.conf

Note that you can already test this in production before merging by briefly setting this setting in the configmap. Talk to me or norman on how to do this